intro-section.tex

```diff
@@ -3,152 +3,272 @@ \section{Cylc Introduction}
-contains a single task named \lstinline{hello_world} which waits 5 seconds
-then outputs the text ``Hello World!'' to standard out. The task is defined in
-in the \lstinline=[runtime]= section and cylc is told to run the task by the
-\lstinline{graph = hello_world} part.
+This suite runs a single task named \lstinline{hello} that prints ``Hello
+World!'' to standard out and then sleeps for 30 seconds before exiting.
 
-\begin{lstlisting}[language=suiterc, title=hello world example]
+% Call task "hello" to distinguish task name from suite name.
+% Sleep for 30 seconds - long enough to be seen in the GUI.
+
+\begin{lstlisting}[language=suiterc]
 [scheduling]
     [[dependencies]]
-        graph = hello_world
+        graph = hello
 [runtime]
-    [[hello_world]]
-        script = sleep 5; echo "Hello World!"
+    [[hello]]
+        script = echo "Hello World!"; sleep 30
 \end{lstlisting}
 
-To run a new suite there are three things that must be done:
-
-\begin{enumerate}
-    \item The suite must be registered with cylc
-    \item (optionally) the suite can be validated to ensure there are no errors
-            in the suite.rc file.
-    \item Finally, we can run the suite.
-\end{enumerate}
-
-To do this we can use the \lstinline[language=bash]{rose suite-run} command.
-This command requires a rose-suite.conf file which should sit alongside the
-suite.rc. The rose-suite.conf file can be empty.
+The \lstinline{[scheduling]} section tells cylc to run the hello task
+immediately on start-up (because it does not depend on other tasks and has no
+clock triggers).  The \lstinline=[runtime]= section configures cylc to 
+run the inlined ``Hello World!'' shell scripting for the task (as a background
+job on the suite host, by default).  Section~\ref{Tasks} below explains how
+this works.
 
-\begin{lstlisting}[language=bash]
-$ touch rose-suite.conf
-$ rose suite-run
-\end{lstlisting}
+\more{This isn't quite the simplest possible cylc suite. You can omit the runtime
+configuration for a task and cylc will automatically create a minimal dummy job
+for it. This can be useful for mocking up suites, and temporarily "dummying
+out" real tasks}.
 
-For this tutorial we will be using the \lstinline{rose suite-run} command
-though these three steps can be undertaken without using rose using the
-following cylc commands.
+Here's how to run a new suite (don't do this yet; see below):
 
 \begin{lstlisting}[language=bash]
 $ cylc register hello_world /path/to/suite/
 $ cylc validate hello_world
 $ cylc run hello_world
 \end{lstlisting}
 
-\paragraph*{Demo}
-To run the hello world example, create a directory called hello\_world. This
-will be our suite directory.
+\more{
+    \begin{itemize}
+        \item
+    \lstinline{cylc-register} just associates a name with a suite definition.
+    Commands that parse a suite can refer to this suite name or the file path;
+    commands that connect to a suite daemon must refer to the suite name.
```

Comment:
This is in a "more info" box, and it's there to explain *why* registration is needed, and what a "suite name" is used for.    If we run through the tutorial without explaining things like this will the users understand what they're doing?    It seems to me, if I was told "you need to register a suite before you can run it", my next thought would be "what does that mean, and why do it need to do it?".